frontend/model/contracts/identity.js

```diff
@@ -2,13 +2,16 @@
 
 import sbp from '@sbp/sbp'
 import Vue from 'vue'
+import { maxLength } from 'vuelidate/lib/validators'
```

Comment:
With the changes coming in #1308, importing a third-party dependency inside of the contracts is no longer non-trivial, as it requires careful thought and inclusion via `common.js`, see #1308 and the comments in that file for details.

To save ourselves some headaches, let's avoid doing that if we can. `maxLength` really isn't needed here anyway, as that's a simple `.length <= ...` operation.